src/client/components/Searchbar.tsx

```diff
@@ -22,11 +22,17 @@ const Searchbar: React.FC<SearchbarProps> = ({
       const result = await findAllShops();
       const cities: SearchbarOptions[] = result.map((e) => {
         return {
-          label: e.city + ", " + e.name + ", " + e.address,
+          label: e.city,
           value: e.city,
         };
       });
-      setOptions(cities);
+      const filteredCities = cities.filter(
+        (elem, index, self) =>
+          self.findIndex((t) => {
+            return t.label === elem.label && t.value === elem.value;
+          }) === index
+      );
```

Comment:
Sorry, I cannot understand what this is doing.
Why are you filtering for?

Reply:
If I dont add that functions, the cities on the searchbar will get duplicated. For example, if we have 5 shops in "Shibuya", and the user inputs "Shibuya", the searchbar display will show "Shibuya" 5 times, instead of just once, as intended. This functions filters the searchbar to display it only once. After the user select the city, then all shops in that city will be displayed.

Reply:
I see!
I think in that case, it may be better to modify the SQL side. (Maybe adding a `findUniqueCities` method? https://knexjs.org/guide/query-builder.html#distinct)
Because, previously, we were getting information on all shops, but with the changes we made, we want to get the name of the city where there are no duplicates.
We should reduce the amount of data exchanged as much as possible, and SQL is very good at processing data!

Reply:
Done!